### PR TITLE
General Cleanup & Improvement

### PR DESCRIPTION
**Pull request by jwisser**
Sorry for the monolithic pull request. Sometime you’ll have to explain
to me the preferred way of handling a bunch of changes like this. In
the meantime:
- lots of cleanup
- a few additions (for example, in the **You never finished "Does my
  character level?"** section)

Also, "make it fit the OMNE purview" in the **Materials** section of the core rulebook is sufficiently unclear that I don't know how to fix it. But it should be fixed.

### CODE REVIEW

**Comment by pcperini** [comment]
Honestly, monolithic PRs are probably just a fact of life in early-stage. Hopefully the need for this will go down as the project mature.

**Comment by pcperini** [comment]
@jwisser Mind opening a proper issue for the `OMNE purview` problem?

File: books/md/Core-Rulebook.md [hunk 1]
```diff
@@ -123,28 +150,29 @@ These categories are broken down by “roll range”. In order to determine roll
...
-> > *You then roll a `d20` (`15`), and add your DEXT (`0`) and Lockpicking (`3`).*
+> > *You then roll a `d20` (`15`), and add your DEXT (`0`) and Lockpicking (`3`). Finally, you add one extra point because you're applying your supporting skill of Locksmithing.*
 >
-> > *Your total is `18`.*
+> > *Your total is `19`.*
 >
 >
 > GM: “You succeed, the lock clicks open revealing...”
 >
 > > *The DC was `16`. Your minimum score was `DEXT + Lockpicking` (`3`).*
```
Comment (pcperini) [inline books/md/Core-Rulebook.md]: If an additional support skill (Locksmithing) was used, there needs to be a corresponding `+1` added to the minimum score.

File: books/md/Core-Rulebook.md [hunk 2]
```diff
@@ -123,28 +150,29 @@ These categories are broken down by “roll range”. In order to determine roll
...
 >
-> > *`16 - 4 = 13`. `12 < 13 ≤ 20`. This lock was of hard difficulty.*
+> > *`16 - 3 = 13`. `12 < 13 ≤ 20`. This lock was of hard difficulty.*
```
Comment (pcperini) [inline books/md/Core-Rulebook.md]: Which would make it

> `16 - 4 = 12`. `1 < 12 ≤ 12`. This lock was of moderate difficulty.

But I might consider changing the DC to, say, 13, to make the math more clear:

> `13 - 4 = 9`. `1 < 9 ≤ 12`.

**Comment by jwisser** [comment]
Math changes made.

**Comment by pcperini** [comment]
Looks great! Thanks.

**Pull request merged**